dali/test/python/test_operator_decoders_image.py

```diff
@@ -356,8 +357,9 @@ def pipe(device):
         decoded = fn.decoders.image_slice(encoded, device=device, end=[10000], axes=[1])
         return decoded
     p = pipe(device)
-    assert_raises(RuntimeError, p.run)
+    p.build()
+    assert_raises(RuntimeError, p.run, glob='*IsInRange*failed*')
```

Comment:
`IsInRange`? Is this a part of the assert or error message itself?

Reply:
There's no message there.
```C++
DALI_ENFORCE(crop.IsInRange(shape));
```
```C++
DALI_ENFORCE(data.roi.IsInRange(dims));
```

Reply:
I would add it then. I though we want to get rid of printing assert itself and leave only error messages.

Reply:
Agreed, we decided to not rely on the DALI_ENFORCE condition, in this case I would introduce a proper message.

Reply:
Agreed... that we shouldn't. What's not necessarily agreed whether this is a part of this task or not.